src/box/box.cc

```diff
@@ -681,6 +681,157 @@ box_check_uri(const char *source, const char *option_name)
 	return 0;
 }
 
+static inline int
+box_check_uri_delimiters(const char *source)
```

Comment:
Please add comment to this func as well. Like

Check that URI does not start or end with one of the listed delimiters.
@param source String containing URI. It supposed to be NULL-terminated.

Reply:
Also I don't get why we check that URI can't start/end with delimiters, so that I'm asking to write comment for it.

Reply:
Add comment which also explain why we need this check.